### ISSUE TITLE
Sync Bookmark with piped

### ISSUE THREAD

**Issue by NeeRaj-2401** [libre-tube/LibreTube#2674]
### Describe your suggested feature

as piped now supports playlists  [in this PR](https://github.com/TeamPiped/Piped/pull/1951)
it will be good to have synced the bookmarks over account.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by NeeRaj-2401** [comment]
i discovered that even piped does not support syncing it over account.
so i'll leave it open hoping that piped will support it soon and libertube too :)

**Comment by ARBoyGo** [comment]
Mmm, That great update, Soon we'll be able do this.... 
I think after we get this update, We no longer need cloning playlist because this is much easy..  still playlist is playlist bookmark is bookmark

**Comment by Bnyro** [comment]
I tend do add features asap when Piped adds support for it, to keep a better overview I prefer to not have issues open for something Piped's API doesn't support yet.

**Comment by ARBoyGo** [comment]
@NeeRaj-2401, First you need to open an issue in piped API, As LibreTube is just doing that part only after they added it.... Good luck 🤞 hope for best
